### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by aliabbani**
In this milestone I:
- Create forms to perform the following functions:
Creates a Post on behalf of the current_user.
Create a comment on behalf of the current_user.
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
```
Comment (rloterh) [inline app/controllers/comments_controller.rb:1]: - [x] Your comments controller does not have a new action. Kindly ensure to improve upon this aspect adding a new action to the comments controller like below:

```ruby
class CommentsController < ApplicationController
  ...

  def new
    @comment = Comment.new
  end
```

File: config/routes.rb [hunk 2]
```diff
@@ -5,5 +5,10 @@
     resources :posts
   end
 
+  resources :posts do
+    resources :comments, only: [:create]
```
Comment (rloterh) [inline config/routes.rb]: - [ ] Kindly ensure that there is a pair of routes in the `config/routes.rb` file for creating comments like below:

```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
   GET   /posts/:post_id/comments/new(.:format)     comments#new
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Abbani, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (aliabbani) [reply app/controllers/comments_controller.rb:1]: Done, thank you!

File: config/routes.rb (hunk 2 above)
Comment (aliabbani) [reply config/routes.rb]: Done, Thank you!

**Review by sinansevgi — APPROVED** [review]
Hi @aliabbani, 

Great Work! :+1: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Happy Hacking! :computer: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**